packages/manager/apps/dedicated/client/app/configuration/configuration.scss

```diff
@@ -0,0 +1,14 @@
+ovh-manager-configuration {
+  @media (max-width: 768px) {
+    .trusted-nic-label {
+      display: block;
+      margin: 0 -1rem;
+      margin-top: -2rem;
+    }
+
+    .oui-badge {
+      border-radius: 0;
+      max-width: none;
+    }
```

Comment:
Can't be achieved with bs4 utilities ?

Reply:
Yeah, it's possible. But I did not realize it at that time. Since the QC is already done, I prefer not to make this change now...